### DISCUSSION TITLE
Is webmesh dormant or complete?

### DISCUSSION THREAD

**Discussion by amano-kenji** [webmeshproj/webmesh#22] [Q&A] [answered]
It seems there has been no new commit for almost one year.

Perhaps, there is nearly perfect already? Or, is it dormant?

**Comment by tinyzimmer** [comment, marked as answer]
> It seems there has been no new commit for almost one year.
> 
> Perhaps, there is nearly perfect already? Or, is it dormant?

It's pretty much dormant for the time being. I had to refocus on jobs. There is a chance I could pick things back up in the future if things settle financially for me.

**Reply by amano-kenji** [reply]
I looked into free open-source alternatives. I found strongswan(IPSec), OpenVPN, and nebula.

Nebula and OpenVPN force rotation of certificates, but certificates allow me to revoke access from specific nodes. strongswan lets me choose between pre-shared secret and certificates.

Headscale has to catch up with tailscale which isn't open-source. Netbird and netmaker restrict free functionalities.

If I'm subject to VPN censorship, then I should probably use any VPN software that's compatible with v2ray.

Between strongswan, openvpn, and nebula, what do you prefer?

**Reply by bmullan** [reply]
@tinyzimmer 
I'm assuming you have been absorbed by other activities than Webmesh but I'd like to ask you your opinion
as to the usability of the last release:

 **[https://github.com/webmeshproj/webmesh/releases/tag/v0.17.1](https://github.com/webmeshproj/webmesh/releases/tag/v0.17.1)**

Of all the Webmesh Documentation, CLI (ie wmctl) etc were their things that did not work or work correctly?
It would be helpful to know before expending time trying to utilize things that might just not be fully implemented yet.

Basically, my question comes down to is, for what is documented,  what is working and what is not?

You appear to have done so much work on Webmesh so thanks.
